Update README.md (#4)
cko-recruitment/payment-gateway-challenge-dotnet: README.md

# Instructions for candidates

> This is an outline of the challenge that we will be completing in your coding interview. During this session, we will work on this brief with you in a pairing style, with you driving the paring. The expectation is not to finish the task, but use the time to work through solving the problem. There is also time dedicated during the interview to be asked questions around approach.

Reviewer: ```suggestion
During this session we'll work using the pair programming technique with you as the driver and we will be the observer or navigator. The expectation is not to finish the task, but use the time to work through solving the problem.
```

Ahead of the session, please **do take a look at the details of this project** and familiarize yourself with the requirements and the documentation.

> Please ensure to fully prepare your development environment for the session. Setup any IDEs and ensure that you have an libraries or dependencies and installed and loaded ahead of your session. We want to avoid spending time in the interviews on setup.

Reviewer: ```suggestion
Please ensure to fully prepare your development environment for the session. Setup any IDEs and ensure that you have all libraries and dependencies installed and ready ahead of your session to avoid losing time on this during the session.
```

> Once your environment is setup, you can begin working on the challenge, to get any initial components built/started. However please do not spend longer than 60 minutes working on this. We want to spend time working with in the session, rather than reviewing what you have already done.

Reviewer: ```suggestion
Once your environment is set up you can begin working on the challenge before the session. Please do not spend longer than 60 minutes working on this as we want to spend time working with in the session rather than reviewing what you have already done.
```
